examples/gh-org-runner-mig-native-simple/README.md

```diff
@@ -0,0 +1,53 @@
+# Example Runners that support Docker Workflows
+
+## Overview
+
+This example showcases how to use startup scripts to deploy organisation runners using the `gh-runner-mig` module.
+
+We use startup/shutdown scripts to install the runner binary, register the runner when it comes online and de-register when shut down.
+
+## Steps to deploy this example
+
+- Step 1: Create terraform.tfvars file with the necessary values.
+
+Access tokens require repo scope for private repos and public_repo scope for public repos. GitHub Apps must have the administration permission to use this API. Authenticated users must have admin access to the repository to use this API.
```

Comment:
I have yet to verify this, but I think the required token permissions should be described as follows:
```suggestion
Access tokens require the `organization_self_hosted_runners` permission to use this API.
```
Can you check which permissions are needed for the token, and update the documentation?

Reply:
Updated https://github.com/terraform-google-modules/terraform-google-github-actions-runners/pull/37/files#diff-fe9b11be3f708962a43495cfbc2fac9e34d23a180fcd8480a9a4682b3a08ac2eR13-R16